driver-service/src/main/java/com/modsen/driverservice/services/AutoServiceImpl.java

```diff
@@ -0,0 +1,92 @@
+package com.modsen.driverservice.services;
+
+import com.modsen.driverservice.dto.AutoDto;
+import com.modsen.driverservice.dto.AutoPageResponse;
+import com.modsen.driverservice.dto.AutoResponseList;
+import com.modsen.driverservice.entities.Auto;
+import com.modsen.driverservice.exceptions.*;
+import com.modsen.driverservice.mappers.AutoMapper;
+import com.modsen.driverservice.repositories.AutoRepository;
+import com.modsen.driverservice.services.interfaces.AutoService;
+import com.modsen.driverservice.util.ExceptionMessage;
+import lombok.RequiredArgsConstructor;
+import org.springframework.data.domain.Page;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+@RequiredArgsConstructor
+public class AutoServiceImpl implements AutoService {
+
+    private final AutoRepository autoRepository;
+
+    private final AutoMapper autoMapper;
+
+    private final PaginationService paginationService;
+
+    public AutoResponseList getAll()
```

Comment:
Помечай методы, которые объявлены в интерфейсе и реализованы в классе аннотацией @Override. Просмотри остальные методы, где нужно добавь
```suggestion
    @Override
    public AutoResponseList getAll() {
```